lib/datadog/tracing/contrib/utils/quantization/http.rb

```diff
@@ -22,6 +22,14 @@ def url(url, options = {})
               options[:placeholder] || PLACEHOLDER
```

Comment:
Can you add unit tests for the changes in this file?

Reply:
Yup, I have them right here, I just detected a last second issue that needed some adjustment.